src/openfermion/utils/_trotter_qasm_conversion.py

```diff
@@ -0,0 +1,331 @@
+#	Licensed under the Apache License, Version 2.0 (the "License");
+#	you may not use this file except in compliance with the License.
+#	You may obtain a copy of the License at
+#
+#		http://www.apache.org/licenses/LICENSE-2.0
+#
+#	Unless required by applicable law or agreed to in writing, software
+#	distributed under the License is distributed on an "AS IS" BASIS,
+#	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#	See the License for the specific language governing permissions and
+#	limitations under the License.
+
+from openfermion.ops import QubitOperator
+import numpy as np
+
+'''
+Description:
+		Approximates the exponential of an operator composed of Pauli
+		matrices, by Trotterization. Outputs QASM format to a python
+		stream.
+
+		Change-of-basis gates:
+		H			: Z to X basis (and back)
+		Rx(pi/2)	: Z to Y basis ..... 
+		Rx(-pi/2) 	: Y to Z basis ..... 
+
+'''
+
+
+
+
+def third_order_trotter_helper(hamiltonian_dict,trotterized_list,op_a, op_b):
+	'''
+	This function recursively trotterizes a QubitOperator according
+		to the scheme:
+		e^(A+B) = e^(7/24 * A) e^(2/3 * B) e^(3/4 * A) e^(-1/24 * A) e^(B)
+
+	Used in trotterize() function.
+
+	Params: 
+		trotterized_list: list that the function populates with trotterized
```

Comment:
I would recommend returning the generated list rather than in-place modification for clarity in a more functional style.  If you think that is totally unwieldy, use at a minimum the convention of separating input parameters from output parameters, making clear which will be modified when the routine is called.